index.js

```diff
@@ -134,6 +135,20 @@ module.exports = function ExpressOpenApi (_routePrefix, _doc, _opts) {
     middleware.document = generateDocument(middleware.document, req.app._router || req.app.router)
     res.json(middleware.document)
   })
+
+  // OpenAPI document as yaml
+  router.get(new RegExp(`${routePrefix}.[(yaml)(yml)]`), (req, res) => {
```

Comment:
Routes can also be specified as an array of paths, I would prefer this over the regexp approach.

```suggestion
  router.get([`${routePrefix}.yaml`, `${routePrefix}.yml`], (req, res) => {
```